### DISCUSSION TITLE
v0.8.1

### DISCUSSION THREAD

**Discussion by mxcl** [teaxyz/teaBASE#44] [Announcements]
**Full Changelog**: https://github.com/teaxyz/teaBASE/compare/v0.8.0...v0.8.1

<hr /><em>This discussion was created from the release <a href='https://github.com/teaxyz/teaBASE/releases/tag/v0.8.1'>v0.8.1</a>.</em>

**Comment by hapsyd** [comment]
how to use? any video that community can see?
